openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/Graph.vue

```diff
@@ -641,7 +641,7 @@ export default {
             if (typeof rawValue === 'string' || isNaN(rawValue)) {
               return 'NaN'
             } else {
-              return rawValue == null ? '--' : rawValue.toFixed(2)
```

Comment:
Comparing against null is the one place where you should use double equals. That will catch both null and undefined.  Probably need to change those back.